features/steps/psu_api_steps.py

```diff
@@ -0,0 +1,27 @@
+import uuid
+
+# pylint: disable=no-name-in-module
+from behave import given, when  # pyright: ignore [reportAttributeAccessIssue]
+from methods.api.psu_api_methods import send_status_update
+from methods.shared.common import get_auth
+from utils.prescription_id_generator import generate_short_form_id
+from utils.random_nhs_number_generator import generate_single
+
+
+@given("I am authorised to send prescription updates")
+def i_am_authorised_to_send_prescription_updates(context):
+    env = context.config.userdata["env"].lower()
+    if "sandbox" in env:
+        return
+    context.auth_token = get_auth(env, "PSU")
+
+
+@when("I send an update")
```

Comment:
should we name this 'I send a collected update' (or similar) so we can have other steps in future to send different updates

Reply:
Turned this into a parameterised step so we can pass in the status and terminal status each time we want to run this step